Shared/Samples/Measure distance in scene/MeasureDistanceInSceneView.swift

```diff
@@ -0,0 +1,183 @@
+
+private extension MeasureDistanceInSceneView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A scene with an imagery basemap.
+        let scene = {
+            let scene = ArcGIS.Scene(basemapStyle: .arcGISTopographic)
+            
+            // Add elevation source to the base surface of the scene with the service URL.
+            let elevationSource = ArcGISTiledElevationSource(url: .brestElevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            
+            // Create the building layer and add it to the scene.
+            let buildingsLayer = ArcGISSceneLayer(url: .brestBuildingsService)
+            scene.addOperationalLayer(buildingsLayer)
+            
+            return scene
+        }()
+        
+        /// An analysis overlay for location distance measurement.
+        let analysisOverlay = AnalysisOverlay()
+        /// The location distance measurement.
+        let locationDistanceMeasurement = LocationDistanceMeasurement(
+            startLocation: Point(x: -4.494677, y: 48.384472, z: 24.772694, spatialReference: .wgs84),
+            endLocation: Point(x: -4.495646, y: 48.384377, z: 58.501115, spatialReference: .wgs84)
+        )
+        
+        /// A measurement formatter for converting the distances to strings.
+        let measurementFormatter: MeasurementFormatter = {
+            let measurementFormatter = MeasurementFormatter()
+            measurementFormatter.unitOptions = .providedUnit
+            measurementFormatter.numberFormatter.minimumFractionDigits = 2
+            measurementFormatter.numberFormatter.maximumFractionDigits = 2
+            return measurementFormatter
+        }()
+        
+        init() {
+            // Set scene to the viewpoint specified by the location distance measurement.
+            let lookAtPoint = Envelope(
+                min: locationDistanceMeasurement.startLocation,
+                max: locationDistanceMeasurement.endLocation
+            ).center
+            let camera = Camera(lookingAt: lookAtPoint, distance: 200, heading: 0, pitch: 45, roll: 0)
+            scene.initialViewpoint = Viewpoint(boundingGeometry: lookAtPoint, camera: camera)
+            
+            // Add location distance measurement to the analysis overlay to display it.
+            analysisOverlay.addAnalysis(locationDistanceMeasurement)
+        }
+    }
+}
+
+private extension String {
+    /// The user instruction message for setting the start location.
+    static let startMessage = "Tap on the map to set the start location."
+    
+    /// The user instruction message for setting the end location.
+    static let endMessage = "Tap and drag on the map to set the end location."
+}
+
+private extension URL {
+    /// A elevation image service URL for Brest, France.
+    static var brestElevationService: URL {
+        URL(string: "https://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer")!
```

Comment:
`common-samples/issues/3852`. If you like you can lend a help hand on figuring out why the data is down and help to replace affected designs. 🙂